packages/teams-js/test/public/mail.spec.ts

```diff
@@ -0,0 +1,325 @@
+import { GlobalVars } from '../../src/internal/globalVars';
+import { app } from '../../src/public/app';
+import { mail } from '../../src/public/mail';
+import { Utils } from '../utils';
+
+describe('mail', () => {
+  // Use to send a mock message from the app.
+  const utils = new Utils();
+
+  beforeEach(() => {
+    utils.processMessage = null;
+    utils.messages = [];
+    utils.childMessages = [];
+    utils.childWindow.closed = false;
+    GlobalVars.frameContext = undefined;
+
+    // Set a mock window for testing
+    app._initialize(utils.mockWindow);
+  });
+
+  afterEach(() => {
+    // Reset the object since it's a singleton
+    if (app._uninitialize) {
+      app._uninitialize();
+    }
+  });
+
+  describe('openMailItem', () => {
+    const openMailItemParams: mail.OpenMailItemParams = {
+      itemId: '',
```

Comment:
It seems like the `openMailItem` function should throw if it gets an empty item id?